idb/postgres/postgres_migrations.go

```diff
@@ -371,14 +371,19 @@ func m4RewardsAndDatesPart1(db *IndexerDb, state *MigrationState) error {
 		`ALTER TABLE account ADD COLUMN rewards_total bigint NOT NULL DEFAULT 0`,
 
 		// created/closed round
+		`ALTER TABLE account ADD COLUMN deleted boolean DEFAULT NULL`,
```

Comment:
This needs to be a new migration because there are systems that have already applied m4

Reply:
I think those systems are out of luck

Reply:
That can't be the answer.
m4 went in 2020-12-18
Release 2.3.0 happened Jan 5. We have to support that release.
Though, https://releases.algorand.com/ is still only showing 2.2.1, so, 2.3.0 never really happened and is just fake garbage in github? Can we delete the tag?

Reply:
Also this could be `DEFAULT FALSE` and simplify some pointers

Reply:
If it is `DEFAULT FALSE` is there a way to know whether the migration has initialized them or not?

Regarding a new migration or not, I'm not sure why beta code is pushed to `master` but at the moment it is. The release notes will have to say something like "this release is not compatible with 2.3.0-beta".

Reply:
It's not just `2.3.0-beta`, I see a `2.3.0` tag which apparently was premature.

Reply:
ah, yeah that would be a problem.

@algojohnlee could you confirm whether or not 2.3.0 has been released?

Reply:
2.3.0 tag was created in anticipation of merge to master, but we held off on actually officially publishing. It would be a problem for anyone who built off master, but not for anyone using our repositories. I would feel okay with not supporting it as an official release, despite the icky merge to master.